newsAnalysis/config.py

```diff
@@ -0,0 +1,9 @@
+import os
```

Comment:
Do we still need a config file if the parameters are already set in the docker-compose?

Reply:
it needs to be loaded somewhere, and calling it config.py gives a clear indication where that happens